lib/tags/index.js

```diff
@@ -18,11 +18,14 @@ export default class TagContainer extends Component {
     const {
       tags,
       removeTag,
-      resetColour
+      resetColour,
+      resetBackgroundColour,
+      resetBorderColour
     } = this.props;
     const resetButtonStyle = {
       color: resetColour,
-      border: '1px solid' + resetColour
+      backgroundColor: resetBackgroundColour,
+      border: '1px solid' + resetBorderColour
     };
```

Comment:
I think that's the way we used to have it but it had been changed. I decided to go with the quickest fix that could be refactored later